vizro-core/src/vizro/actions/_action_loop/_action_loop.py

```diff
@@ -37,5 +41,8 @@ def _build_actions_models():
             List of required components for each `Action` in the `Dashboard` e.g. list[dcc.Download]
 
         """
-        actions = _get_actions_on_registered_pages()
-        return html.Div([action.build() for action in actions], id="app_action_models_components_div", hidden=True)
+        return html.Div(
+            [action.build() for action in cast(Iterable[Action], model_manager._get_models(Action))],
```

Comment:
Is this this casting done just for mypy?

Reply:
Yes, `cast` is only ever required for mypy. In these cases actually it feels like mypy should be able to work this out for itself and not need any casting at all, but unfortunately that doesn't work, not sure why.